x-pack/plugins/security_solution/kibana.json

```diff
@@ -33,6 +33,7 @@
   "requiredBundles": [
     "kibanaUtils",
     "esUiShared",
-    "kibanaReact"
+    "kibanaReact",
+    "ingestManager"
```

Comment:
this is a new feature from kibana platform, see https://github.com/elastic/kibana/pull/70911

This allows us to still load the shared code that we import from `ingestManager` without the plugin actually being loaded.  So, since `"ingestManager"` is optional we can either load it or not and the Administration section will handle accordingly.

There are existing tests that already checked the behavior of the Admin section with or without ingestManager either within Space or if the plugin is loaded at all.